blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/Containers.java

```diff
@@ -117,9 +116,7 @@ public Link getLink() {
 
             @Override
             public T get(String name) {
-                T u = base.get(name);
-                if (u==null)    return null;
-                return u;
+                return base.get( name);
```

Comment:
I can see this is a good idea, but this PR is huge already. Perhaps restrict changes to guava-related for this PR? 

```suggestion
                return base.get(name);
```

Reply:
this code was hurting my eyes so I change it. no a big problem as far I can see